core/src/main/java/apoc/periodic/Periodic.java

```diff
@@ -177,7 +180,7 @@ private long executeNumericResultStatement(@Name("statement") String statement,
     }
 
     private void validateQuery(String statement) {
-        Util.validateQuery(db, statement);
+        Util.validateQuery(db, statement, READ_WRITE, WRITE, READ_ONLY);
```

Comment:
non va bene perché questo varrebbe per tutte le periodic, o sbaglio?

Reply:
Sì per com'è fatto vale per tutte infatti ho messo il test anche per la iterate in quanto tutte hanno `Mode.WRITE`.
Però certo, potrei mettere questa cosa solo nella `commit`